# sensor board mesage as an array of range_n_robot_detector
range_n_robot_detector[] range_n_bearing
